src/metabase/lib/schema/expression/temporal.cljc

```diff
@@ -78,17 +78,18 @@
 (mr/def ::timezone-id
   [:and
    ::common/non-blank-string
-   [:or
-    (into [:enum
-           {:error/message "valid timezone ID"
-            :error/fn      (fn [{:keys [value]} _]
-                             (str "invalid timezone ID: " (pr-str value)))}]
-          (sort
-           #?(;; 600 timezones on java 17
-              :clj (ZoneId/getAvailableZoneIds)
-              ;; 596 timezones on moment-timezone 0.5.38
-              :cljs (.names (.-tz moment)))))
-    ::literal/string.zone-offset]])
+  ;;  [:or
+  ;;   (into [:enum
+  ;;          {:error/message "valid timezone ID"
+  ;;           :error/fn      (fn [{:keys [value]} _]
+  ;;                            (str "invalid timezone ID: " (pr-str value)))}]
+  ;;         (sort
+  ;;          #?(;; 600 timezones on java 17
+  ;;             :clj (ZoneId/getAvailableZoneIds)
+  ;;             ;; 596 timezones on moment-timezone 0.5.38
+  ;;             :cljs (.names (.-tz moment)))))
```

Comment:
This line made the frontend change, I think it doesn't find the timezones when this is evaluated 🤷